specification/attestation/resource-manager/Microsoft.Attestation/stable/2020-10-01/attestation.json

```diff
@@ -525,6 +525,14 @@
         "attestUri": {
           "type": "string",
           "description": "Gets the uri of attestation service"
+        },
+        "privateEndpointConnections": {
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "../../../../../common-types/resource-management/v1/privatelinks.json#/definitions/PrivateLinkServiceConnectionState"
```

Comment:
Per my comment in last time, here should be PrivateEndpointConnection,  not PrivateLinkServiceConnectionState. right?